resources/profiles/Flashforge/filament/Flashforge AD4 Generic ABS.json

```diff
@@ -0,0 +1,64 @@
+{
+  "type": "filament",
+  "filament_id": "GFB99",
+  "setting_id": "GFSA04",
+  "name": "Flashforge AD4 Generic ABS",

```

Comment:
Can you rename all filaments(name and file name) to something like `Generic ABS @Flashforge AD4`?